packages/front-end/services/experiments.ts

```diff
@@ -180,3 +181,19 @@ export function useDomain(
   });
   return [lowerBound || 0, upperBound || 0];
 }
+
+export function applyMetricOverrides(
+  metric: MetricInterface,
+  experiment: ExperimentInterfaceStringDates
+): boolean {
+  if (!metric) return false;
+  const metricOverride = experiment?.metricOverrides?.find(
+    (mo) => mo.id === metric.id
+  );
+  if (metricOverride) {
+    metric.conversionWindowHours = metricOverride.conversionWindowHours;
+    metric.conversionDelayHours = metricOverride.conversionDelayHours;
+    return true;
```

Comment:
We don't want to mutate metrics on the front end. That will definitely break things since we share the same objects in a global context. We could get away with this on the back end, but not on the front end.